internal/os/exec/exec.go

```diff
@@ -0,0 +1,8 @@
+package exec
```

Comment:
If we're thinking about moving this out into somewhere public, maybe "execpath" might be a good name for this package, with Look as the name of the function. Doesn't matter for now though; just a thought.

Reply:
Done. Nice change too! Thanks